Makefile

```diff
@@ -57,7 +57,7 @@ unit-test-nocoverage:
 unit-test-nocoverage-host: golint-host govet-host
 	HOST_TEST=1 GOGC=1000 godep go test -v ./... -check.v
 
-build: golint
+build: golint govet
```

Comment:
This seems to have sneaked in. Would you be OK with having this in a separate PR which makes more changes to the makefile? I don't have a strong opinion about this, but we should avoid this when possible.